src/promptflow/promptflow/_sdk/entities/_connection.py

```diff
@@ -618,6 +626,23 @@ def _get_schema_cls(cls):
         return FormRecognizerConnectionSchema
 
 
+class CustomStrongTypeConnection(_Connection):
```

Comment:
Please add docstring to this class, and may hint user do not use it directly, give the custom strong type define example or doc link.

Reply:
docstring and example added